### ISSUE TITLE
Duplicate EFOs in Curation App

### ISSUE THREAD

**Issue by earlEBI** [EBISPOT/goci#795]
![Screenshot 2022-07-19 at 15.01.57.png](https://images.zenhubusercontent.com/5e95cce7cc796e84ca614069/77a591db-6ccb-4104-879d-260340e9fe14)

EFO duplicates keep appearing on this page in the curation app: 
http://ves-ebi-e4-x.ebi.ac.uk:8080/gwas/curation/efotraits

I have already cleaned this up last week but they have reappeared. 
Again I have cleaned up as listed here
[https://app.zenhub.com/files/2995118/bc27da0f-3555-4c27-92a2-a78db601462d/download](https://app.zenhub.com/files/2995118/bc27da0f-3555-4c27-92a2-a78db601462d/download)

However, where 'deleted' is listed, these changes don't seem to have been saved as when you search for the term, eg 'creatine measurement', you can still see the duplicates in curation app. 

For the terms where two different URIs existed, I deleted the newer term from the depocuration as well. 

Otherwise, duplicates, eg. the obsolete MONDO term for Myopia, do not show in Depocuration app, but I think will still cause problems when importing.

**Comment by earlEBI** [comment]
Please delete the following EFO from depocuration as I am uanble to because they say they're linked to studies. I changed the EFOs in Curation app:

Creatine measurement	EFO_0021579
Inositol measurement	EFO_0021606
Theobromine measurement	EFO_0021673

**Comment by ljwh2** [comment]
@earlEBI is this an outstanding issue?

**Comment by earlEBI** [comment]
Yes, although I now see the EFOs are currently linked to PMID 33634981. So I'll try and change them first.

**Comment by earlEBI** [comment]
Actually, the problem terms are not in the Catalog UI, so this does still need to be cleaned up by a developer. 

So, for instance:
If you search for "not in use" (which is what I added to the 3 redundant term labels" on the studies page of depocuration, you see three linked studies from PMID 33634981.
GCST90129098
GCST90129130
GCST90129205
<img width="1422" alt="Screenshot 2023-05-16 at 19 11 05" src="https://github.com/EBISPOT/goci/assets/61545198/6619a410-ac45-42c8-9def-ee7827394c55">

But when you search for these GCSTs in the live UI, they are linked to different (correct) EFOs.

**Comment by earlEBI** [comment]
I don't think this is an issue anymore. I think the inability to delete EFOs mentioned above is covered by this [ticket](https://app.zenhub.com/workspaces/gwas-59df823c4a6feb3786810391/issues/gh/ebispot/goci/1488). 
I think this ticket can be closed.

**Comment by ljwh2** [comment]
I don't think the new ticket will handle existing traits, only process going forward, so will keep open for now.
